app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -139,6 +161,7 @@ const ExperimentDetails: React.FC = () => {
         <p>
           The results can be found here: <a href="#">(link here)</a>
```

Comment:
Tangentially related to the above "(link here)" debate, but we might want to make this link text/format consistent (either in this PR or I can file a new issue once this gets merged).